test/base/resource_getters.go

```diff
@@ -14,56 +14,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// resource_getters.go contains functions which get actual resources given the meta resource.
```

Comment:
I wouldn't put the name of the file. Just say something like "This file contains...."

Reply:
Done.